scripts/submit_jobs.py

```diff
@@ -66,6 +67,7 @@ def generate_ntuple_config(configtype, newtag):
 PARSER.add_argument('-d', '--dataset', required=True)
 PARSER.add_argument('-o', '--outputsite', required=True)
 PARSER.add_argument('-n', '--no_exec')
+PARSER.add_argument('-c', '--caloparams')
```

Comment:
The `customise` command at [L50](https://github.com/cms-hcal-trigger/Validation/pull/4/files#diff-d2f446b0e82f32b776ed125168954504R50) should only be used if this argument is defined. It should no longer be necessary to explicitly specify the CaloParams, as they are now taken from the global tag, so [L50](https://github.com/cms-hcal-trigger/Validation/pull/4/files#diff-d2f446b0e82f32b776ed125168954504R50) would override the CaloParams from the GT. You should check that you get exactly identical results if the CaloParams are specified explicitly or if they are taken from the global tag.

Reply:
Could you please follow up on this comment?

Reply:
Sorry, somehow I missed that you had already implemented the requested changes.